flash/video/classification/data.py

```diff
@@ -119,6 +213,7 @@ def __init__(
         val_transform: Optional[Dict[str, Callable]] = None,
         test_transform: Optional[Dict[str, Callable]] = None,
         predict_transform: Optional[Dict[str, Callable]] = None,
+        label_field: str = "ground_truth",
```

Comment:
I bet the Flash team will object to a FiftyOne-specific parameter appearing here. It's probably best to replace `label_field` with `**kwargs` or ``**data_source_kwargs`` and document that these are source-specific keyword arguments that are passed to the `DataSource`.